lib/bikeshed/bikeshed-writer.ts

```diff
@@ -23,15 +25,25 @@ export async function writeBikeshed(
     outputStream.on("error", reject);
   });
 
-  writeMetadata(model.metadata, outputStream);
-  writeIntroduction(model, outputStream);
-  writeDataModel(model.schemas, outputStream);
+  constructBikeshed(model, outputStream);
   outputStream.end();
 
   return result;
 }
 
-function writeMetadata(content: Record<string, string>, stream: WriteStream) {
+export function getBikeshed(model: Bikeshed): string {
+  const stream = new StringWriteStream();
```

Comment:
Based on the https://github.com/opendata-mvcr/model-driven-data/tree/develop#inputoutput-as-a-parameter
i would pass the StringWriteStream as a parameter. As now the caller get back a string every time. The string is constructed using the StringWriteStream. 

I think it would be cleaner to pass the StringWriteStream, or it's interface, as an argument. The function then return nothing, just write the content into the StringWriteStream.